### PR TITLE
Correct defined but not used errors in StoryViewer.js and index.js. 

### PR DESCRIPTION
**Pull request by rhiannonstanford**
This PR corrects the following defined but not used errors:

./src/components/pages/AudioBook/StoryViewer.js
Line 3:10: 'Header' is defined but never used - deleted
Line 8:10: 'connect' is defined but never used - deleted
Line 12:10: 'tasks' is defined but never used - deleted
Line 17:10: 'hasViewedAllPages' is assigned a value but never used - this component level state variable will be used as the component is further developed.  Left as is.
Line 69:9: 'onFinish' is assigned a value but never used - this helper function will be used as the component is further developed. Left as is.

./src/index.js
Line 40:10: 'Modal' is defined but never used - deleted
Line 104:9: 'authHandler' is assigned a value but never used - authHandler will be used as the component is further developed.  Resolved errors through the use of a console.log within the helper function.  Will not result in terminal log until authHandler gets implemented.  

Resolves Trello card: https://trello.com/c/hbMR04zI/605-defined-but-never-used-h
PR video walkthrough: https://www.youtube.com/watch?v=ir5qHxuMQP0

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -98,13 +97,13 @@ ReactDOM.render(
...
 function App() {
   // The reason to declare App this way is so that we can use any helper functions we'd need for business logic, in our case auth.
-  // React Router has a nifty useHistory hook we can use at this level to ensure we have security around our routes.
   const history = useHistory();
 
   const authHandler = () => {
-    // We pass this to our <Security /> component that wraps our routes.
-    // It'll automatically check if userToken is available and push back to login if not :)
+    // We pass this function to our <Security /> component that wraps our routes.
+    // Checks if userToken is available and pushes back to login if not
```
Comment (rhiannonstanford) [inline src/index.js:104]: Comment clean up to make more readable.

File: src/index.js [hunk 2]
```diff
@@ -98,13 +97,13 @@ ReactDOM.render(
...
     history.push('/login');
+    console.log('AuthHandler', authHandler);
```
Comment (rhiannonstanford) [inline src/index.js:106]: The addition of a console.log here resolves the defined but not used error related to authHandler.  This auth function will be used as this component is further developed.  This change will not produce a console.log in the terminal until the authHandler function gets implemented.

**Review by rhiannonstanford — COMMENTED** [review]
Self review

File: src/index.js (hunk 1 above)
Comment (artofmayhem) [reply src/index.js:104]: Agreed on commenting clean up. Love to see the thought process though.

**Review by artofmayhem — APPROVED** [review]
Overall nice bit of code cleanup with a quick and easy method of cleaning up the defined but unused error by creating a console log to use without just commenting it out.

**Review by Yonathan-Admasu728 — APPROVED** [review]
great fixes  all over. excellent attention to detail to leave hasviewedAllpages as it is.  also  very thoughtful  to console the AuthHandler in the index component.

**Comment by OniWarrior** [comment]
I agree with all the changes. Many of the things that I'm seeing in the components are imports that are never used.  It seems a bit careless to import stuff you don't need. This was a very good cleanup. It makes the code base more understandable because we understand exactly what's being used.

File: src/index.js (hunk 2 above)
Comment (kendra4227) [reply src/index.js:106]: Looks good.

File: src/index.js [hunk 3]
```diff
@@ -98,13 +97,13 @@ ReactDOM.render(
 
 function App() {
   // The reason to declare App this way is so that we can use any helper functions we'd need for business logic, in our case auth.
```
Comment (Julianisrel) [inline src/index.js:99]: Good explanation for function use.

**Review by Julianisrel — APPROVED** [review]

**Review by just-kato — APPROVED** [review]

**Comment by just-kato** [comment]
Ready for merge

File: src/components/pages/AudioBook/StoryViewer.js [hunk 4]
```diff
@@ -1,15 +1,11 @@
 import React, { useState, useEffect, useCallback } from 'react';
 import { Document, Page, pdfjs } from 'react-pdf';
-import { Header } from '../../common';
```
Comment (BryanPAnderson) [inline src/components/pages/AudioBook/StoryViewer.js:3]: Correct defined but not used errors.

File: src/components/pages/AudioBook/StoryViewer.js [hunk 5]
```diff
@@ -1,15 +1,11 @@
...
 import { useAuth0 } from '@auth0/auth0-react';
 import { Button } from 'antd';
 import { ArrowLeftOutlined, ArrowRightOutlined } from '@ant-design/icons';
 import { SizeMe } from 'react-sizeme';
-import { connect } from 'react-redux';
```
Comment (BryanPAnderson) [inline src/components/pages/AudioBook/StoryViewer.js:8]: Correctly imported but not used.

File: src/components/pages/AudioBook/StoryViewer.js [hunk 6]
```diff
@@ -69,7 +65,6 @@ const StoryViewer = props => {
   const onFinish = e => {
     markAsRead(user, props.tasks.id);
     push('/child/mission-control');
-
```
Comment (BryanPAnderson) [inline src/components/pages/AudioBook/StoryViewer.js:72]: Delete half closing tag ">".

File: src/components/pages/AudioBook/StoryViewer.js [hunk 7]
```diff
@@ -1,15 +1,11 @@
...
-import { Header } from '../../common';
 import { useAuth0 } from '@auth0/auth0-react';
 import { Button } from 'antd';
 import { ArrowLeftOutlined, ArrowRightOutlined } from '@ant-design/icons';
 import { SizeMe } from 'react-sizeme';
-import { connect } from 'react-redux';
 import { useHistory } from 'react-router-dom';
-
 import { markAsRead } from '../../../api';
-import { tasks } from '../../../state/actions';
```
Comment (BryanPAnderson) [inline src/components/pages/AudioBook/StoryViewer.js:12]: Not using import.

**Review by BryanPAnderson — COMMENTED** [review]
reviewed code line by line, 4 import errors, and 1 syntax error.

**Pull request merged**